libs/base/ui/overlays/notification-center/src/notification-center.component.ts

```diff
@@ -43,6 +45,16 @@ export class NotificationCenterComponent
     `;
   }
 
+  protected resolveContent(
```

Comment:
what about supporting content directly inside i18n directive?

Reply:
Added it to the i18nmixin, due to it is components-related stuff